docs_user/modules/proc_configuring-a-ceph-backend.adoc

```diff
@@ -87,11 +77,11 @@ stringData:
     mon_host = 10.1.1.2,10.1.1.3,10.1.1.4
 ----
 
-. Configure `extraMounts` within the `OpenStackControlPlane` CR:
+. In your `OpenStackControlPlane` CR, configure the {Ceph} back end from your {OpenStackShort} deployment by using the `extraMounts` key. For example:
```

Comment:
Do customers need to adjust the `extraMounts` configuration to the values in their environment? That is, is the code in this step just an example, or are they supposed to use these exact values?

Reply:
They can use the following snippet as it is.
The only difference might be in the `propagation` list, which depends on the services that are backed by Ceph.
The snippet assumes the ceph.conf and keyring are propagated everywhere.
Note that this part that uses `extraMounts` is used to inject both `ceph.conf` and `ceph.client.openstack.keyring` to the Pods, but this is not providing the ceph backend configuration for each service: that part is present and defined for each service in their own adoption section.

Reply:
RHOSO Pods is generic enough to let the reader resolve it into a `CR`, a `Service`, a `Component` within a `Service`, a `Container`, etc.
To be consistent with the `extraMounts` wording, I would say "inject `ceph.conf` and `ceph.client.openstack.keyring` to the `OpenStack services` defined in the `propagation` list".
@klgill the propagation list is clearly visible to the example below, and contains the services that need those files because they're supposed to be configured with Ceph.